sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt

```diff
@@ -171,7 +169,15 @@ internal class RadarLocationManager(
 
     internal fun handleBootCompleted() {
         logger.d("Handling boot completed")
+        wakeUp()
+    }
+
+    internal fun handleTaskRemoved() {
+        logger.d("Handling task removed")
+        wakeUp()
```

Comment:
intuitively, it seems a little odd that we're "waking up" when the task is removed - maybe it's just the fn name or my ignorance, but would the SDK need to wake up if it was already active when the app was backgrounded

Reply:
Good call on this one. I tested the changes without this wakeup, and it works fine. The real trick was removing the flag in the manifest file.